src/components/Layout/Header/NavBar/UserMenu.tsx

```diff
@@ -160,7 +160,6 @@ export const UserMenu: React.FC<{ onClick?: () => void }> = memo(({ onClick }) =
   const { isConnected, isDemoWallet, walletInfo, connectedType, isLocked, isLoadingLocalWallet } =
     state
 
-  if (isLocked) disconnect()
```

Comment:
This was a bandaid, confirmed with @reallybeard we don't want this anymore, though we may if the app is borked when MM is locked, to be tested